scripts/dts/extract/default.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2018 Bobby Noelte
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from extract.globals import *
+from extract.directive import DTDirective
+
+##
+# @brief Manage directives in a default way.
+#
+class DTDefault(DTDirective):
+
+    def __init__(self):
+        pass
+
+    ##
+    # @brief Extract directives in a default way
+    #
+    # @param node_address Address of node owning the clockxxx definition.
```

Comment:
I guess this comment is not accurate

Reply:
Yes should read clocks (clockxxx is for future use).